CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ configure_file(mender-version.h.in mender-version.h)
 
 project(mender)
 
+# The CMake documentation is quite ambiguous on the subject of available build types, and I haven't
```

Comment:
https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#id38

Reply:
How did I not find that the first time around?? I was looking all over the place... Thanks! :open_mouth: